.github/ISSUE_TEMPLATE/join_org.yml

```
@@ -0,0 +1,19 @@
+name: Join Organization
+description: This issue template can be helpful for beginner who wish to join the community..
+title: "Invite me to the organization"
```

This is just an idea; I'm not sure if it's good, though.

```suggestion
title: "[ORG] Invite me to the organization"
```